Core/Src/sensors/steering.c

```diff
@@ -15,54 +15,63 @@
 
 extern volatile uint16_t ADC_DMA_buffer[ADC_LINES];
 
+// calculation of current angle based on current adc reading, maximum steering wheel angle
+// and from the min and max reading in the calibration using Thales' theorem
+double calculation_steering(double current_read) {
+    double steering_calc;
+    steering_calc = ANG_MAX_STEERING
+                    * ((current_read - MIN_STEERING) / (MAX_STEERING - MIN_STEERING));
+    return (uint16_t)steering_calc;
+}
```

Comment:
```suggestion
uint16_t calculation_steering(double current_read) {
    double steering_calc;
    steering_calc = ANG_MAX_STEERING
                    * ((current_read - MIN_STEERING) / (MAX_STEERING - MIN_STEERING));
    return (uint16_t)steering_calc;
}
```
Tipos do return incoerentes